test/converters/converters_test.go

```diff
@@ -0,0 +1,166 @@
+					{
+						Name:      "testFailMe",
+						ClassName: "_TtCC17rtgtrghtrgUITests17rtgtrghtrgUITests18rtgtrghtrg3UITests",
+						Time:      0.09049093723297119,
+						Failure:   "file:///Users/tamaspapik/Develop/ios/rtgtrghtrg/rtgtrghtrgUITests/rtgtrghtrgUITests.swift:CharacterRangeLen=0&EndingLineNumber=67&StartingLineNumber=67 - XCTAssertTrue failed",
+					},
+					{
+						Name:      "testLaunchPerformance",
+						ClassName: "_TtCC17rtgtrghtrgUITests17rtgtrghtrgUITests18rtgtrghtrg3UITests",
+						Time:      0.036438941955566406,
+					},
+
+					{
+						Name:      "testExample()",
+						ClassName: "rtgtrghtrg2UITests",
+						Time:      0.06093001365661621,
+					},
+					{
+						Name:      "testFailMe()",
+						ClassName: "rtgtrghtrg2UITests",
+						Time:      0.08525991439819336,
+						Failure:   "file:///Users/tamaspapik/Develop/ios/rtgtrghtrg/rtgtrghtrgUITests/rtgtrghtrgUITests.swift:CharacterRangeLen=0&EndingLineNumber=104&StartingLineNumber=104 - XCTAssertTrue failed",
+					},
+					{
+						Name:      "testLaunchPerformance()",
+						ClassName: "rtgtrghtrg2UITests",
+						Time:      0.041545987129211426,
+					},
+
+					{
+						Name:      "testExample()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.07126104831695557,
+					},
+					{
+						Name:      "testExample2()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.043392062187194824,
+					},
+					{
+						Name:      "testFailMe()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.04290807247161865,
+					},
+					{
+						Name:      "testFailMe2()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.08395206928253174,
+						Failure:   "file:///Users/tamaspapik/Develop/ios/rtgtrghtrg/rtgtrghtrgUITests/rtgtrghtrgUITests.swift:CharacterRangeLen=0&EndingLineNumber=129&StartingLineNumber=129 - XCTAssertTrue failed",
+					},
+					{
+						Name:      "testLaunchPerformance()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.048184990882873535,
+					},
+					{
+						Name:      "testLaunchPerformance2()",
+						ClassName: "rtgtrghtrg4UITests",
+						Time:      0.030627012252807617,
+					},
+
+					{
+						Name:      "testExample()",
+						ClassName: "rtgtrghtrgUITests",
+						Time:      0.030849933624267578,
+					},
+					{
+						Name:      "testFailMe()",
+						ClassName: "rtgtrghtrgUITests",
+						Time:      0.030683040618896484,
+					},
+					{
+						Name:      "testLaunchPerformance()",
+						ClassName: "rtgtrghtrgUITests",
+						Time:      0.03138589859008789,
+					},
+				},
+			},
+		},
+	}
+
+	for _, test := range []struct {
+		name          string
+		converter     Intf
+		testFilePaths []string
+		wantDetect    bool
+		wantXML       junit.XML
+		wantXMLError  bool
+	}{
+		{
+			name:          "xcresult3",
+			converter:     &xcresult3.Converter{},
+			testFilePaths: []string{filepath.Join(os.Getenv("BITRISE_SOURCE_DIR"), "_tmp/xcresults/xcresult3_multi_level_UI_tests.xcresult")},
```

Comment:
Can you add the descriptions of test cases (e.g. any specific setup like inherited classes) used in this xcresult, or any info needed to produce a similar xcresult file.